src/TabNine.ts

```diff
@@ -195,5 +203,29 @@ export class TabNine {
       })
     });
   }
+}
+
+export class TabNineExtension {
+  private extension: vscode.Extension<any>;
+  
+  private constructor(ext: vscode.Extension<any>) {
+    this.extension = ext;
+  }
+
+  getExtensionPath(): string {
```

Comment:
you can defined them as getters: `get version() {}` and then call them as properties `ext.version`

Reply:
cool, thanks 👍